server/chart_config.json

```diff
@@ -160,6 +159,12 @@
       "denominator": "Count_Household"
     }
   },
+  {
+    "category": ["Health", "Life Expectancy"],
+    "title": "Life Expectancy",
+    "statsVars": ["LifeExpectancy_Person"],
+    "isOverview": true
```

Comment:
this is somewhat out of date now. could you please copy over relatedChart as well?

Reply:
please copy over the relatedChart object from the existing config. and post-merge from upstream, we no longer require the second item in the category array.

Reply:
done